includes/blocks/class-sensei-block-helpers.php

```diff
@@ -94,18 +94,19 @@ public static function render_style_attributes( $class_names, $css ) {
 	 * Add default style to list of classes if no style is selected. If a parent classname is supplied, it will override
 	 * the default style.
 	 *
-	 * @param array $attributes        Block attributes.
-	 * @param array $parent_attributes Parent block attributes.
+	 * @param array  $attributes         Block attributes.
+	 * @param array  $parent_attributes  Parent block attributes.
+	 * @param string $default_style_name Default style name.
 	 *
 	 * @return string
 	 */
-	public static function block_class_with_default_style( $attributes, $parent_attributes = [] ) {
+	public static function block_class_with_default_style( $attributes, $parent_attributes = [], $default_style_name = 'default' ) {
```

Comment:
This is not needed for this PR anymore, but as it can be useful for different situations, I kept it.